### PR TITLE
CSS STYLES

### PR DESCRIPTION
**Pull request by gchen1196**
npm install
npm run react-dev
npm run server-dev

localhost:3002

### CODE REVIEW

File: client/src/components/App.jsx [hunk 1]
```diff
@@ -1,32 +1,31 @@
...
-  //     url: '/house/110',
-  //   })
-  //     .then((house) => {
-  //       this.setState({pictures: house.data.pictures}, () => console.log('set state', house.data.pictures));
-  //     });
-  // }
+  componentDidMount() {
+    axios({
+      method: 'get',
+      url: '/house/115',
```
Comment (gregoryinouye) [inline client/src/components/App.jsx]: should you make this more general now? i assume you don't want to hardcode this any more?

File: client/src/components/App.jsx [hunk 2]
```diff
@@ -1,32 +1,31 @@
...
+    })
+      .then((house) => {
+        console.log(house)
```
Comment (gregoryinouye) [inline client/src/components/App.jsx]: if not needed any more, please remove any extra console.logs

File: client/src/components/App.jsx [hunk 3]
```diff
@@ -1,32 +1,31 @@
...
+        this.setState({pictures: house.data.pictures}, () => console.log('set state', house.data.pictures));
```
Comment (gregoryinouye) [inline client/src/components/App.jsx]: extra console.log?

File: client/src/components/Modal.jsx [hunk 4]
```diff
@@ -95,24 +105,22 @@ class Modal extends React.Component {
...
       <div className="modal" tabIndex="0" onKeyDown={e => this.onModalKeyPress(e)}>
         <div className="x-button" onClick={e => this.onButtonClick(e)}>x</div>
         <div className="right-button" onClick={e => this.onRightClick(e)}>{rightButton}</div>
         <div onClick={e => this.onLeftClick(e)} className="left-button">{leftButton}</div>
-        <div className='main-container'>
-          <div className='container-center'>
-            <ModalPhoto url={this.state.currentUrl} />
-            <Description isVerified={this.state.currentPicture.isVerified} position={this.state.currentPosition + 1} size={this.state.length0 + 1} description={this.state.currentPicture.description} />
-            <div className={`photo-carousel active-${this.state.currentPosition}`} style={{ opacity: 1 }}>
-              <div className="photo-carousel-wrapper" style={{ transform: `translateX(-${this.state.currentPosition * (100 / pictures.length)}%)` }}>
```
Comment (gregoryinouye) [inline client/src/components/Modal.jsx]: if you want to leave this alone to move on to the proxy server stuff that's fine, but i think that the style stuff should really go with the rest of the css you have if possible. was there a reason you kept this styling separate?

File: client/src/components/Modal.jsx [hunk 5]
```diff
@@ -95,24 +105,22 @@ class Modal extends React.Component {
...
     return (
       <div className="modal" tabIndex="0" onKeyDown={e => this.onModalKeyPress(e)}>
         <div className="x-button" onClick={e => this.onButtonClick(e)}>x</div>
         <div className="right-button" onClick={e => this.onRightClick(e)}>{rightButton}</div>
         <div onClick={e => this.onLeftClick(e)} className="left-button">{leftButton}</div>
-        <div className='main-container'>
-          <div className='container-center'>
-            <ModalPhoto url={this.state.currentUrl} />
-            <Description isVerified={this.state.currentPicture.isVerified} position={this.state.currentPosition + 1} size={this.state.length0 + 1} description={this.state.currentPicture.description} />
-            <div className={`photo-carousel active-${this.state.currentPosition}`} style={{ opacity: 1 }}>
```
Comment (gregoryinouye) [inline client/src/components/Modal.jsx]: if you want to leave this alone to move on to the proxy server stuff that's fine, but i think that the style stuff should really go with the rest of the css you have if possible. was there a reason you kept this styling separate?

**Comment by gchen1196** [comment]
Thanks Greg for going through my code line by line. I know this was a huge PR. I will refactor my code and make the necessary changes.

**Pull request merged**